src/makieplotting.jl

```diff
@@ -307,6 +307,95 @@ function Makie.plot!(AR::Arrows{<:Tuple{<:MakiePlotter{dim}}}) where dim
     Makie.arrows!(AR, ps, ns, arrowsize=AR[:arrowsize], normalize=AR[:normalize], colormap=AR[:colormap], color=lengths, lengthscale=AR[:lengthscale])
 end
 
+"""
+    elementinfo(ip::Interpolation; kwargs...)
+    elementinfo(cell::AbstractCell; kwargs...)
+    elementinfo(ip::Type{Interpolation}; kwargs...)
+    elementinfo(cell::Type{AbstractCell}; kwargs...)
+
+- `plotnodes::Bool=true` plots the nodes as circles/spheres
+- `strokewidth::Int=2` how thick faces/edges are drawn
+- `color::Symbol=theme(scene,:linecolor)` color of the faces/edges and nodes
+- `markersize::Int=30` size of the nodes
+- `nodelables=false` global node id labels
+- `nodelabelcolor=:darkblue`
+- `textsize::Int=15` size of the label's text
+"""
+@recipe(Elementinfo) do scene
+    Attributes(
+    plotnodes=true,
+    strokewidth=2,
+    color=theme(scene, :linecolor),
+    markersize=30,
+    textsize=60,
+    nodelabels=true,
+    nodelabelcolor=:darkred,
+    nodelabeloffset=(0.0,0.0),
+    facelabels=true,
+    facelabelcolor=:darkgreen,
+    facelabeloffset=(0,0),
+    edgelabels=true,
+    edgelabelcolor=:darkblue,
+    edgelabeloffset=(-40,-40),
+    )
+end
+
+function Makie.plot!(Ele::Elementinfo{<:Tuple{<:Ferrite.Interpolation{dim,refshape}}}) where {dim,refshape}
+    ip = Ele[1][]
+    elenodes = Ferrite.reference_coordinates(ip) |> x->reshape(reinterpret(Float64,x),(dim,length(x)))'
+    dim > 2 ? (lines = Point3f[]) : (lines = Point2f[])
+    _faces = Ferrite.faces(ip)
+    if dim == 2
+        append!(lines, [elenodes[e,:] for boundary in _faces for e in boundary[1:2]]) # 1:2 because higher order node in the middle
+    else
+        _edges = Ferrite.edges(ip)
+        order = Ferrite.getorder(ip)
+        #TODO remove the index monstrosity below after edges are defined consistently see https://github.com/Ferrite-FEM/Ferrite.jl/issues/520
+        append!(lines, [elenodes[e,:] for boundary in _edges for e in boundary[1:((refshape == Ferrite.RefCube) ? 1 : (order > 1 ? 2 : 1)):((refshape == Ferrite.RefCube) ? 2 : end)]]) # 1:2 because higher order node in the middle
+    end
+    boundaryentities = dim == 2 ? _faces : _edges
+    #plot element boundary
+    Makie.linesegments!(Ele,lines,color=Ele[:color], linewidth=Ele[:strokewidth])
+    for (id,face) in enumerate(_faces)
```

Comment:
maybe rename face -> face_nodes (or similar) or add some comment what is happening, because this is not straight forward to understand for potential contributors.